scripts/import-mentors.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * Use this script to import the mentors from the
+ * https://github.com/Coding-Coach/find-a-mentor repository to the mongo
+ * database.
+ * 
+ * Usage:
+ *  $ yarn import:mentors
+ */
+import * as dotenv from 'dotenv';
+import * as fetch from 'node-fetch';
+import * as mongoose from 'mongoose';
+
+import { UserSchema } from './../src/modules/users/schemas/user.schema';
+import { Role } from 'src/modules/users/interfaces/user.interface';
+
+const fetchMentors = async (request: RequestInfo): Promise<any> => {
+  return new Promise(resolve => {
+    fetch(request)
+      .then(response => response.json())
+      .then(body => {
+        resolve(body);
+      });
+  });
+};
+
+async function importMentors() {
+  console.log("Fetch the mentors")
+  const mentors = await fetchMentors('https://raw.githubusercontent.com/Coding-Coach/find-a-mentor/master/src/mentors.json');
+  console.log(`Fetched ${mentors.length} mentors`)
+
+  console.log("Connect to database")
+  mongoose.connect(process.env.MONGO_DATABASE_URL, { useNewUrlParser: true });
+
+  var User = mongoose.model('User', UserSchema);
+
+  console.log("Store mentors to database")
+  for (const mentor of mentors) {
+    console.log(`Process mentor '${mentor.id}'`)
+
+    var user = new User({
```

Comment:
Before creating a new user make sure the user doesn't exist in the database, maybe the current mentor is already there (while I was testing I got my username twice in the database)

Reply:
Very good point. The new code first checks whether the mentor is already in the database.